### ISSUE TITLE
ReferenceError: document is not defined

### ISSUE THREAD

**Issue by rmoehn** [probmods/webppl-viz#82]
When I want to run a WebPPL program with webppl-viz, I get `ReferenceError: document is not defined`. This problem is already described in issue #79. I'll just add my workaround here, so that people can find it more easily.

Assuming that you've installed all your WebPPL stuff in `~/.webppl`, run this:

```bash
for p in $(find ~/.webppl -name canvas |grep 'node_modules/canvas$' |sed 's/node_modules\/canvas//'); do
    npm --prefix=$p uninstall canvas && npm --prefix=$p install canvas@1.6.4
done
```
This sets all occurrences of canvas to 1.6.4.

And sorry if my Bash is inelegant…

**Comment by longouyang** [comment]
Thanks for the workaround -- I just merged #79 so I'll close this issue. (And apologies for the delay -- just got back from vacation)

**Comment by rmoehn** [comment]
Thanks! And no worries; vacations are important.
